module5/lessons/Week2/Resumes.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: Technical Resumes
+---
+
+The content for the Resume session can be found in [this](https://docs.google.com/presentation/d/1rzswJmrdkNodLgxqq0-YmGhc5Kn23gKhHp_HH0UeoXQ/edit?usp=sharing) Google Slides Presentation.

```

Comment:
phew - this is a lot of slides... wondering if you think we have time in an hour to do this?  Or should this PD be longer?

Reply:
It is still long and I cut a ton! My thought is up until the actually making the Resume part will take roughly 40 minutes. Then they will have a bit of time to get started on actually making their Resume, but mostly they will work on that during office hours/PD worktime on Thursdays. The schedule is a bit unfortunate because that Thursday they have the project not office hours/worktime. So I'm thinking of giving them a week and a half to finish the Resume.

Although, might actually be better to full on switch the weeks for Resumes and GithubPD/Your Story Revisit. Since I don't think that will take more work time. What do you think?

Reply:
Megan McMahon
  [2 minutes ago](https://turingschool.slack.com/archives/C056BR0CKK9/p1695827947447149?thread_ts=1695827230.732079&cid=C056BR0CKK9)
I think I do get notifications... but I also get so many github notifications that things can easily get lost :see_no_evil: .
I agree that with the project, they will probably need more time for the resume deliverable... we could switch the calendar... or, we could have the Resume and github deliverables due at the same time?
This gives them more time for resumes.  I want the github profile to be a bit more time-boxed because this could definitely be a time-suck if given too much time between the session and due date.  Also, this might be the first time we have asked them to manage two deliverables at the same time - so maybe a good organizational challenge for them?





Zoe Farrell
  [< 1 minute ago](https://turingschool.slack.com/archives/C056BR0CKK9/p1695828083777689?thread_ts=1695827230.732079&cid=C056BR0CKK9)
Oooo I can see that being good growth opportunity. Have the two of them due the next week so that the Github Profile is more timeboxed and they still have time for making their resume. Let's try that!